src/utils.js

```diff
@@ -42,7 +42,13 @@ export function get<T: any>(uri: string, options: any): Promise<T> {
             return;
         }
 
-        const opts = Object.assign({ json: true }, options);
+        const opts = Object.assign(
+            {
+                json: false,
+                token: null,
```

Comment:
Maybe just allow sending a `headers` option instead that gets merged into the request options object.